src/components/SearchHint/SearchHint.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import './SearchHint.scss';
+
+export const SearchHint: React.FC<{ hint: string }> = ({ hint }) => (
```

Comment:
We use Interfaces everywhere, why make this an exception? I think it would look better with interface